sr_edc_launch/sr_edc.launch

```diff
@@ -18,8 +18,6 @@
        In case we only have one hand this parameter will normally be "", meaning that the desired params are in the root of those directories
        If it's not "" then it must be followed by a "/" e.g. "hand_2/"  -->
   <arg name="config_dir" default=""/>
```

Comment:
yeah you're right, but in my next PR I am removing this parameter so I guess I missed it :-/